learn-responsive-web-design-by-building-a-piano/index04.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<body>
+
+ <div id="piano">
+     <div class="keys"></div>
+ </div>
+ 
+</body>
+</html>
+
+<!-- 
+    The piano will have 3 sets of 7 white keys and 5 black keys
```

Comment:
This could be broken down a little more. Since we want these projects to work offline, we don't need to include outside links -- a simple explanation is enough. First, have the student create one `<div>`:

"Create a new `<div>` element inside of the `keys` `<div>`."

Then have them add the `class` attribute `key`.

Explain a bit about the piano keys and have the student create the missing `key` `<div>` elements:

"The piano will have 3 sets of 7 white keys and 5 black keys on large displays. You already have one key in this set, so create six more `key` `<div>` elements just below it."